src/service/state.js

```diff
@@ -44,7 +44,7 @@ module.exports = (statePersistence, municipalityService, seService) => {
      * Get information about an strategic ecosystem in an state. Includes:
      * - percentage of the given strategic ecosystem respect the national area
      *
-     * @param {String} stateId environmental authority id
+     * @param {String} stateId state id
```

Comment:
stateId is a number

Reply:
done

Reply:
But it still says "String", please fix it before merging